rhasspynlu/jsgf.py

```diff
@@ -15,15 +15,15 @@ class Substitutable:
     # Names of converters to apply after substitution
     converters: typing.List[str] = field(default_factory=list)
 
-    @classmethod
-    def parse_substitution(cls, sub_text: str) -> typing.Union[str, typing.List[str]]:
+    @staticmethod
+    def parse_substitution(sub_text: str) -> typing.Union[str, typing.List[str]]:
```

Comment:
was there any reason for this to be a classmethod?

Reply:
Nope, I just had forgotten about static methods :/